src/java/htsjdk/samtools/CRAMFileReader.java

```diff
@@ -121,8 +142,26 @@ public CRAMFileReader(final File cramFile, final ReferenceSource referenceSource
         getIterator();
     }
 
+    /**
+     * Create a CRAMFileReader from an input stream and optional index stream using the supplied reference source.
+     *
+     * @param inputStream      index file to be used for random access. Can not be null.
```

Comment:
This can't be right -- `inputStream` here is for the cram file itself, not the index, right?